src/main/java/sawkiewicz/task1/ItemController.java

```diff
@@ -0,0 +1,60 @@
+
+package sawkiewicz.task1;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import javax.annotation.PostConstruct;
+import java.util.HashMap;
+import java.util.Map;
+
+@Controller
+public class ItemController {
+    Map<Character, Integer> items = new HashMap<>();
+
+    @PostConstruct
+    public void init() {
+        items.put('A', 0);
```

Comment:
again, items definition should come from `ItemRepository`, for now in memory implementation is fine